plugin/trino-hudi/src/test/java/io/trino/plugin/hudi/TestHudiPlugin.java

```diff
@@ -44,16 +48,18 @@ public void testCreateConnector()
 
     @Test
     public void testCreateTestingConnector()
```

Comment:
We could just delete this test ... I don't know why we are testing the test code